src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -2070,9 +2080,16 @@ std::pair<std::shared_ptr<Pane>, std::shared_ptr<Pane>> Pane::_Split(SplitState
     //   Move the new guid, control into the second.
     _firstChild = std::make_shared<Pane>(_profile, _control);
     _firstChild->_connectionState = std::exchange(_connectionState, ConnectionState::NotConnected);
+    _secondChild = newPane;
+
+    // If we want the new pane to be the first child, swap the children
+    if (splitType == SplitState::Up || splitType == SplitState::Left)
+    {
+        std::swap(_firstChild, _secondChild);
+    }

```

Comment:
Wow this is really the entire meat of this PR isn't it. So simple

Reply:
10 lines of actual code changes, 200 lines of boilerplate and refactoring :)

Reply:
I forgot about all of the unit tests that needed to be updated, so that is an extra 200 lines of refactoring changes.